documentation/modules/configuring/proc-removing-volumes-from-jbod-storage.adoc

```diff
@@ -21,6 +21,8 @@ IMPORTANT: To avoid data loss, you have to move all partitions before removing t
 
 . Reassign all partitions from the disks which are you going to remove.
 Any data in partitions still assigned to the disks which are going to be removed might be lost.
++
+TIP: Cruise Control is an effective tool for reassigning partitions.
```

Comment:
Actually, Cruise Control currently cannot help with this. This would be another situation where the reassign toll might need to be used.

Reply:
Right. I've updated the note